src/components/NavigationButton.tsx

```diff
@@ -15,7 +15,7 @@ function NavigationButton({
   };
 
   return (
-    <Button variant="contained" onClick={handleClick}>
+    <Button data-cy={`${label.toLowerCase()}-button`} variant="contained" onClick={handleClick}>
```

Comment:
that might be a bit risky as a cypress selector, no? What if we change the label, should that break the tests? Not sure what the best solution here is. Maybe numeric identifiers for the views you can navigate to? That might change less frequently

Reply:
hmmm what would that look like?

Reply:
Good question. I guess the goal would be to have persistent and unique identifiers for each page/view you can navigate to. Numeric identifiers are easy to make unique and persistent, but they are hard to read and debug. So maybe just a string identifier like `column_annotation` and `multi_column_annotation` and so on. Presumably such a list would have to live in the central store of the app. 

But maybe let's discuss this first in an issue and then make a decision and implement.

Reply:
Yea, let's do that cause I'm not sure if it's a good idea to keep that in the store. Could you please open the issue?